src/test/java/com/github/psambit9791/jdsp/TestExport.java

```diff
@@ -0,0 +1,91 @@
+package com.github.psambit9791.jdsp;
```

Comment:
Probably not required if the **io** module is used

Reply:
Fixed in the latest commits